billing-backup-results/src/main/java/com/abiquo/billing/backup/database/BillingBackupResultsDatabase.java

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (C) 2008 - Abiquo Holdings S.L. All rights reserved.
+ *
+ * Please see /opt/abiquo/tomcat/webapps/legal/ on Abiquo server
+ * or contact contact@abiquo.com for licensing information.
+ */
+package com.abiquo.billing.backup.database;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Map;
+
+import org.apache.ibatis.jdbc.ScriptRunner;
+import org.yaml.snakeyaml.Yaml;
+
+public class BillingBackupResultsDatabase
+{
+    private static final String SYSTEM_ZONE_ID = ZoneId.systemDefault().getId();
```

Comment:
I dont know if this can cause a problem, but I think in kinton we save the dates in UTC,here you are using system default or using the now method of localdate time. Please just check if this is a problem.
https://github.com/abiquo/billing-tools/pull/1/files#diff-07a5057c484a896b8ec7f87c364d4a40cead39a451064de7dfff14633bbfc73aR66

https://github.com/abiquo/billing-tools/pull/1/files#diff-07a5057c484a896b8ec7f87c364d4a40cead39a451064de7dfff14633bbfc73aR157

https://github.com/abiquo/billing-tools/pull/1/files#diff-07a5057c484a896b8ec7f87c364d4a40cead39a451064de7dfff14633bbfc73aR175

Reply:
not a problem, same aproach used:
https://github.com/abiquo/rubrik-plugin/blob/3e6dd5dd8d6aacb53d810d53aa17b2e266a6801c/rubrik-billing-backup/src/main/java/com/abiquo/rubrik/billing/database/RubrikBillingDatabase.java#L108-L109